presto-main/src/test/java/com/facebook/presto/metadata/TestFunctionRegistry.java

```diff
@@ -77,7 +75,7 @@ public void testExactMatchBeforeCoercion()
         boolean foundOperator = false;
         for (SqlFunction function : registry.listOperators()) {
             OperatorType operatorType = unmangleOperator(function.getSignature().getName());
-            if (operatorType == OperatorType.CAST || operatorType == OperatorType.SATURATED_FLOOR_CAST) {
+            if (operatorType == CAST || operatorType == OperatorType.SATURATED_FLOOR_CAST) {
```

Comment:
hmmmm... Keep one with `OperatorType` and the other omitted may not be proper?

Reply:
lol i removed both in a later version. Just wait.